torch/_inductor/autotune_process.py

```diff
@@ -371,19 +305,24 @@ def target(self, choice: TritonTemplateCaller) -> float:
         assert self.processes is not None
 
         process = self.processes.get()
```

Comment:
Do we need to worry about matching the GPU to the kernel being tested? i.e. if I have a machine with both an A100 and an H100 and I'm making a kernel for the H100 it's going to be a problem if I pull an A100 out of the queue, won't it?